Description: Remove dep on valuable until rust-valuable-serde makes it through new
Author: John Goerzen <jgoerzen@complete.org>
Last-Update: 2022-10-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,16 +30,5 @@
 version = "1"
 
 [features]
-valuable = ["valuable_crate", "valuable-serde", "tracing-core/valuable"]
-[target."cfg(tracing_unstable)".dependencies.valuable-serde]
-version = "0.1.0"
-optional = true
-default_features = false
-
-[target."cfg(tracing_unstable)".dependencies.valuable_crate]
-version = "0.1.0"
-optional = true
-default_features = false
-package = "valuable"
 [badges.maintenance]
 status = "experimental"
